salt/orch/update-etc-hosts.sls

```diff
@@ -0,0 +1,23 @@
+update_grains:
+  salt.function:
+    - tgt: 'roles:kube-(master|minion)'
+    - tgt_type: grain_pcre
+    - name: saltutil.refresh_grains
+
+update_mine:
+  salt.function:
+    - tgt: 'roles:kube-(master|minion)'
+    - tgt_type: grain_pcre
+    - name: mine.update
+    - require:
+      - salt: update_grains
+
+etc_hosts_setup:
+  salt.state:
+    - tgt: 'roles:kube-(master|minion)'
+    - tgt_type: grain_pcre
+    - concurrent: True
+    - sls:
+      - etc-hosts
```

Comment:
Isn't there any alternative to this? I don't think that restarting all the API servers after a Salt minion starts is a good idea... So, do we really need to force the `eth0` address? I think nobody will ever know about master's addresses other than this one, so I'm not sure we really need to advertise it.